aiohttp_session/__init__.py

```diff
@@ -8,7 +8,34 @@
 from collections.abc import MutableMapping
 
 from aiohttp import web
-
+from aiohttp.web_middlewares import _Handler, _Middleware
+
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Hashable,
+    Iterator,
+    Optional,
+    Tuple,
+)
+
+if TYPE_CHECKING:  # pragma: no cover
+    from mypy_extensions import TypedDict
```

Comment:
Please use `typing_extensions` library instead.
Also, it doesn't need to be guarded by `TYPE_CHECKING` IMHO.

Reply:
Hadn't seen the `typing_extensions` lib up to now. With this it makes sense to not need the `TYPE_CHECKING` guard :)